src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -2,21 +2,14 @@
 
 import org.usfirst.frc.team467.robot.Autonomous.ActionGroup;
 
-import com.ctre.phoenix.motorcontrol.ControlMode;
-
-import edu.wpi.first.wpilibj.XboxController;
-
 public class DriverStation {
 	
 	XBoxJoystick467 driverJoy;
 	
 	private static DriverStation station;
 	
 	// Mapping of functions to Controller Buttons for normal operation
-	// TODO: Create enum for buttons
-	
-	
-
+	// TODO: Create enumerate for buttons
```

Comment:
I don't understand what this TODO is telling me needs to be done. Please add more explanation.